plugins/dev-tools/src/screenshot.js

```diff
@@ -58,7 +58,7 @@ function workspaceToSvg_(workspace, callback, customCss) {
   // Go through all text areas and set their value.
   const textAreas = document.getElementsByTagName('textarea');
   for (let i = 0; i < textAreas.length; i++) {
-    textAreas[i].innerHTML = textAreas[i].value;
+    textAreas[i].innerText = textAreas[i].value;
```

Comment:
Not sure where exactly the conflict was, but I think this is supposed to be `textContent` instead of `innerText` based on https://github.com/google/blockly-samples/pull/2328#issuecomment-2066987077

not sure if it did get updated and that was lost in the merge conflict, or if it never did.